graph/src/data/query/error.rs

```diff
@@ -252,6 +252,12 @@ impl From<StoreError> for QueryExecutionError {
     }
 }
 
+impl From<anyhow::Error> for QueryExecutionError {
```

Comment:
It seems weird that any `anyhow::Error` gets translated into a `QueryExecutionError::StoreError`

Reply:
Super weird, refactored this away.